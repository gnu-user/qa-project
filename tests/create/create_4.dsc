TITLE
    Creating a User (Invalid Username, invalid name)
DESCRIPTION
    Run the login transaction using an admin account. Initiate the create 
    transaction. Enter an invalid username (which contains “END “) and verify 
    that the username is rejected and that an error occurs.
DEPENDENCIES
    login Test Suite
    create_1
