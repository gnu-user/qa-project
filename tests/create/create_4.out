Welcome.
Current user accounts file read successfully.
Available tickets file read successfully.
> 
Enter username: 
Username accepted.
> 
Enter the username to create: 
Invalid username, username cannot be "END".
> 